app/src/main/java/com/windowmirror/android/controller/MainActivity.java

```diff
@@ -267,7 +267,7 @@ public void onEntryCreated(final Entry entry) {
 
         Log.d ("butterfly", "Calling API:");
 
-        Recording recording = entry.toRecording();
```

Comment:
Not related to removing the play button... But I don't see a reason for this unused variable.